raml-maven-plugin/pom.xml

```diff
@@ -23,6 +23,7 @@
                 <artifactId>maven-plugin-plugin</artifactId>
                 <version>3.4</version>
                 <configuration>
+                    <skip>false</skip>
```

Comment:
Can we be more specific as to what are we skipping?  From the branch name, it appears to be "code generation", but i couldn't figure out from the pom.